mne/viz/tests/test_epochs.py

```diff
@@ -125,13 +125,16 @@ def test_plot_epochs():
         plt.close('all')
 
 
-def test_plot_image_epochs():
+def test_plot_epochs_image():
     """Test plotting of epochs image
     """
     import matplotlib.pyplot as plt
     epochs = _get_epochs()
-    plot_image_epochs(epochs, picks=[1, 2])
+    plot_epochs_image(epochs, picks=[1, 2])
     plt.close('all')
+    with warnings.catch_warnings(record=True):
+        plot_image_epochs(epochs, picks=[1, 2])
+        plt.close('all')
```

Comment:
usually we do e.g. `with ... as w: ... assert_true(len(w) >= 1)`, but I'll let it slide this time :)